ember-flight-icons/tests/dummy/app/templates/engineering.hbs

```diff
@@ -1,10 +1,50 @@
 {{page-title "Engineering"}}
-<h2 class="ds-h2">For Engineers</h2>
-<p class="ds-p">
-    See the
-    <a
-      class="ds-a"
-      href="https://github.com/hashicorp/flight/blob/main/ember-flight-icons/README.md"
-    >README</a>
-    for usage instructions.
-  </p>
\ No newline at end of file
+<h2 class="ds-h2">Consumer instructions</h2>
+<p>There are multiple ways to use these icons in your codebase.</p>
+
+<div class="ds-info">
+<p class="ds-p">🚨 Note: All of the flight-icon npm packages are currently in beta and not intended for production use.</p>
+</div>
+
+<h3 class="ds-h3" id="ember-flight-icons">Use in Ember Apps</h3>
+<h4 class="ds-h4">Installation</h4>
+<p class="ds-p">To install, run
+<code class="ds-code">yarn add @hashicorp/ember-flight-icons</code>
+</p>
+
+<h4 class="ds-h4">Understanding the component</h4>
+
+<p class="ds-p">The component comes with the following defaults:
+<ul class="ds-ul">
+  <li class="ds-li">fill (set to currentColor)</li>
+  <li class="ds-li">a unique, automatically generated id</li>
+  <li class="ds-li">aria-hidden (set to true)</li>
+  <li class="ds-li">default size of 16x16 (px)</li>
+  <li class="ds-li">CSS class (flight-icon & icon-NAME)</li>
+  <li class="ds-li">data-test-icon attribute (for test purposes)</li>
+</ul>
+</p>
+
+<p class="ds-p">This makes the required invocation quite terse- <kbd class="ds-kbd">@name</kbd> is the only property that requires specification.
+
+<br>So this invocation:
```

Comment:
suggestion, non-blocking: might want an extra br

```suggestion
<br><br>So this invocation:
```

fwiw i think this looks shippable as-is!

Reply:
I changed the CSS a little bit instead 👍